Makefile

```diff
@@ -4,6 +4,7 @@ BINDIR ?= /usr/local/bin
 IMAGE_REPOSITORY ?= quay.io/cilium/hubble
 CONTAINER_ENGINE ?= docker
 TARGET=hubble
+VERSION=0.6.1-dev
```

Comment:
this is the only modification i made (from 0.7.0-dev => 0.6.1-dev).

Reply:
LGTM